network/gossip/libp2p/test/testUtil.go

```diff
@@ -0,0 +1,191 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/phayes/freeport"
+	"github.com/rs/zerolog"
+	mock2 "github.com/stretchr/testify/mock"
```

Comment:
Goland auto-rename? Probably want to alias `/module/mock` to `module` or `mockmodule`